rand_pcg/src/pcg64.rs

```diff
@@ -40,6 +40,35 @@ pub struct Lcg64Xsh32 {
 pub type Pcg32 = Lcg64Xsh32;
 
 impl Lcg64Xsh32 {
+    /// Multi-step advance functions (jump-ahead, jump-back)
```

Comment:
We should probably mention that this is equivalent to `delta` times calling `next_u32`.